ios-baseUITests/ios_baseUITests.swift

```diff
@@ -159,4 +109,57 @@ class ios_baseUITests: XCTestCase {
     
     XCTAssert(signInButton.isEnabled)
   }
+  
+  // swiftlint:disable line_length
```

Comment:
why is this rule disabled here?

Reply:
We have an swiftlint error there 

`Line Length Violation: Line should be 100 characters or less; currently it has 104 characters (line_length)`